ci/matrix.xml

```diff
@@ -174,7 +174,8 @@
   <task id="tools-cli-stubs"><![CDATA[
     setJavaVersion $java
     sbt ++$scala tools/package ir/test tools/test cli/package cli/assembly \
-        stubs/package jsEnvsTestSuite/test \
+        stubs/package jsEnvsTestSuite/test phantomJSEnv/test \
```

Comment:
Though now that I've already extracted PhantomJS in the other repo, I think I'll just drop that change.